components/typography/metadata/typography.yml

```diff
@@ -1,182 +1,45 @@
-        <p class="spectrum-Body3">Body3 Text <em>Body3 Emphasis</em> <strong>Body3 Strong</strong>.</p>
-        <p class="spectrum-Body4">Body4 Text <em>Body4 Emphasis</em> <strong>Body4 Strong</strong>.</p>
-        <p class="spectrum-Body5">Body5 Text <em>Body5 Emphasis</em> <strong>Body5 Strong</strong>.</p>
-        <p class="spectrum-Detail">Detail Text <em>Detail Emphasis</em> <strong>Detail Strong</strong>.</p>
-      </div>
-      <br/>
-      <div class="spectrum-Article">
-        <p class="spectrum-Body1">Article Body1 Text <em>Article Body1 Emphasis</em> <strong>Article Body1 Strong</strong>.</p>
-        <p class="spectrum-Body2">Article Body2 text <em>Article Body2 Emphasis</em> <strong>Article Body2 Strong</strong>.</p>
-        <p class="spectrum-Body3">Article Body3 Text <em>Article Body3 Emphasis</em> <strong>Article Body3 Strong</strong>.</p>
-        <p class="spectrum-Body4">Article Body4 Text <em>Article Body4 Emphasis</em> <strong>Article Body4 Strong</strong>.</p>
-        <p class="spectrum-Body5">Article Body5 Text <em>Article Body5 Emphasis</em> <strong>Article Body5 Strong</strong>.</p>
-        <p class="spectrum-Detail">Detail Text <em>Detail Emphasis</em> <strong>Detail Strong</strong>.</p>
-      </div>
-  - id: heading-1
-    name: Article
-    description: Typography elements paired to display clear content hierarchies in long form articles.
-    markup: |
-      <div class="spectrum-Article">
-        <h1 class="spectrum-Heading1">Heading 1</h1>
-        <p class="spectrum-Body1">Body copy that pairs with heading 1</p>
-
-        <h2 class="spectrum-Heading2">Heading 2</h1>
-        <p class="spectrum-Body2">Body copy that pairs with heading 2</p>
-
-        <h3 class="spectrum-Heading3">Heading 3</h1>
-        <p class="spectrum-Body3">Body copy that pairs with heading 3</p>
+    status: Verified
+    description: |
+      Body is primarily used for Spectrum components and for blocks of text.
 
-        <h4 class="spectrum-Heading4">Heading 4</h4>
-        <p class="spectrum-Body4">Body copy that pairs with heading 4</p>
+      [View the Body typography component](typography-body.html)
+    markup: |
+      <p class="spectrum-Body spectrum-Body--M">Spectrum is based on real-world situations.</p>
+  - id: detail-m
+    status: Verified
+    name: Detail
+    description: |
+      Detail is used for disclosing extra information or smaller items in hierarchical relationships of text.
 
-        <h5 class="spectrum-Heading5">Heading 5</h5>
-        <p class="spectrum-Body5">Body copy that pairs with heading 5</p>
-      </div>
-  - id: heading-1
+      [View the Detail typography component](typography-detail.html)
+    markup: |
+      <p class="spectrum-Detail spectrum-Detail--XL">Our recommendations</p>
+  - id: code-m
     name: Typography - Code
-    description: Typographic styles for computer code.
+    status: Verified
+    description: |
+      Code is used for text that represents code.
```

Comment:
Maybe check with Jess if this sounds good?

Reply:
This is copied directly from the Spectrum website.